deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -57,7 +57,35 @@ spec:
       {{- end }}
       {{- if $options.hasConfigMap }}
       shareProcessNamespace: true
+      {{- end }}
       initContainers:
+      - image: {{ include "nvidia-device-plugin.fullimage" . }}
+        name: gpu-feature-discovery-imex-init
+        command: ["/bin/bash", "-c]
+        args:
+        - |
+          IMEX_NODES_CONFIG_FILE=/etc/nvidia-imex/nodes_config.cfg
+          if [[ -f /config/${IMEX_NODES_CONFIG_FILE} ]]; then
+            echo "Removing cached IMEX nodes config"
+            rm -f /config/${IMEX_NODES_CONFIG_FILE}
+          fi
+
+          if [[ ! -f /driver-root/${IMEX_NODES_CONFIG_FILE} ]]; then
+            echo "No IMEX nodes config path detected; Skipping"
+            exit 0
+          fi
+
+          echo "Copying IMEX nodes config"
+          mkdir -p $(dirname /config//config/${IMEX_NODES_CONFIG_FILE})
+          cp /driver-root/${IMEX_NODES_CONFIG_FILE} /config/${IMEX_NODES_CONFIG_FILE}
+        volumeMounts:
+        - name: config
+          mountPath: /config
+        - name: driver-root
+          mountPath: /driver-root/etc
+          subPath: etc
+          readOnly: true
+      {{- if $options.hasConfigMap }}
```

Comment:
Why was this line added?

Reply:
It wasn't added, it was essentially moved so that the config init container is guarded by this conditional. Note that the sharedprocessNamespace is also guarded by the same option.

Reply:
Okay, I see it now. Thanks.